### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Fix linter warnings

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: check_process [hunk 1]
```diff
@@ -17,14 +15,9 @@
 		setup_private=1
 		setup_public=1
 		upgrade=1
-		upgrade=1	from_commit=bd2c87a6fd4212771f9c06e7965598a5ea744ffc
```
Comment (yalh76) [inline check_process:20]: you should keep a test upgrade from previous version

File: scripts/install [hunk 2]
```diff
@@ -80,20 +87,18 @@ ynh_add_fpm_config
 ynh_script_progression --message="Securing files and directories..."
 
 # Set permissions to app files
+chown -R $app:www-data $final_path
 chmod -R 775 $final_path
```
Comment (yalh76) [inline scripts/install]: You should remove `-R` to `chmod`

File: scripts/install [hunk 3]
```diff
@@ -107,4 +112,4 @@ ynh_systemd_action --service_name=nginx --action=reload
 # END OF SCRIPT
 #=================================================
 
-ynh_script_progression --message="Installation of $app completed"
+ynh_script_progression --message="Installation of $app completed" --last
```
Comment (yalh76) [inline scripts/install]:
```suggestion
ynh_script_progression --message="Installation of $app completed"
```

File: scripts/remove [hunk 4]
```diff
@@ -45,8 +45,16 @@ ynh_script_progression --message="Removing PHP-FPM configuration..."
...
+
+# Delete a system user
+ynh_system_user_delete --username=$app
+
 #=================================================
 # END OF SCRIPT
 #=================================================
 
-ynh_script_progression --message="Removal of $app completed"
+ynh_script_progression --message="Removal of $app completed" --last
```
Comment (yalh76) [inline scripts/remove]:
```suggestion
ynh_script_progression --message="Removal of $app completed"
```

File: scripts/restore [hunk 5]
```diff
@@ -42,10 +42,18 @@ test ! -d $final_path \
...
 #=================================================
-ynh_script_progression --message="Restoring the nginx configuration..."
+ynh_script_progression --message="Restoring the NGINX configuration..."
 
 ynh_restore_file --origin_path="/etc/nginx/conf.d/$domain.d/$app.conf"
 
+#=================================================
+# RECREATE THE DEDICATED USER
+#=================================================
+ynh_script_progression --message="Recreating the dedicated system user..." --weight=1
```
Comment (yalh76) [inline scripts/restore]:
```suggestion
ynh_script_progression --message="Recreating the dedicated system user..."
```

File: scripts/restore [hunk 6]
```diff
@@ -42,10 +42,18 @@ test ! -d $final_path \
...
+
+# Create the dedicated user (if not existing)
+ynh_system_user_create --username=$app --home_dir="$final_path"
```
Comment (yalh76) [inline scripts/restore]:
```suggestion
ynh_system_user_create --username=$app
```

File: scripts/restore [hunk 7]
```diff
@@ -59,7 +67,9 @@ ynh_restore_file --origin_path="$final_path"
 ynh_script_progression --message="Restoring user rights..."
 
 # Restore permissions on app files
-chown -R www-data:www-data $final_path
+chown -R $app:www-data $final_path
+chmod -R o-rwx $final_path
+chmod -R 775 $final_path
```
Comment (yalh76) [inline scripts/restore]:
```suggestion
chmod 775 $final_path
```

File: scripts/restore [hunk 8]
```diff
@@ -82,4 +92,4 @@ ynh_systemd_action --service_name=nginx --action=reload
 # END OF SCRIPT
 #=================================================
 
-ynh_script_progression --message="Restoration completed for $app"
+ynh_script_progression --message="Restoration completed for $app" --last
```
Comment (yalh76) [inline scripts/restore]:
```suggestion
ynh_script_progression --message="Restoration completed for $app"
```

File: scripts/upgrade [hunk 9]
```diff
@@ -107,8 +112,8 @@ ynh_add_fpm_config
 ynh_script_progression --message="Securing files and directories..."
 
 # Set permissions on app files
+chown -R $app:www-data $final_path
 chmod -R 775 $final_path
```
Comment (yalh76) [inline scripts/upgrade]:
```suggestion
chmod 775 $final_path
```

File: scripts/upgrade [hunk 10]
```diff
@@ -121,4 +126,4 @@ ynh_systemd_action --service_name=nginx --action=reload
 # END OF SCRIPT
 #=================================================
 
-ynh_script_progression --message="Upgrade of $app completed"
+ynh_script_progression --message="Upgrade of $app completed" --last
```
Comment (yalh76) [inline scripts/upgrade]:
```suggestion
ynh_script_progression --message="Upgrade of $app completed"
```

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Review by yalh76 — APPROVED** [review]

**Comment by yalh76** [comment]
!testme

**Pull request merged**